src/app/zap-templates/templates/app/enums.zapt

```diff
@@ -3,18 +3,20 @@
 // Prevent multiple inclusion
 #pragma once
 
+#include <stdint.h>
+
 // ZCL enums
 
 {{#zcl_enums}}
 {{#unless (isStrEqual label "Status")}}
 {{#unless (isStrEqual label "ReportingDirection")}}
 
 // Enum for {{label}}
-typedef enum {
+enum EmberAf{{asType label}} : uint32_t{
```

Comment:
How difficult is it to actually make this output the right width type?

Reply:
#7394 it will be fixed later